gitlabHelper.js

```diff
@@ -0,0 +1,79 @@
+import * as settings from './settings';
+
+export default class GitlabHelper {
+  constructor(gitlabApi, gitlabSettings) {
+    this.gitlabApi = gitlabApi;
+    this.gitlabUrl = gitlabSettings.url;
+    this.gitlabToken = gitlabSettings.token;
+    this.gitlabProjectId = gitlabSettings.projectId;
+  }
+
+  /**
+   * List all projects that the GitLab user is associated with.
+   */
+  async listProjects() {
+    try {
+      const projects = await this.gitlabApi.Projects.all({ membership: true });
+
+      // print each project with info
+      for (let i = 0; i < projects.length; i++) {
+        console.log(
+          projects[i].id.toString(),
+          '\t',
+          projects[i].name,
+          '\t--\t',
+          projects[i].description
+        );
+      }
+
+      // instructions for user
+      console.log('\n\n');
+      console.log(
+        'Select which project ID should be transported to github. Edit the settings.js accordingly. (gitlab.projectID)'
```

Comment:
The settings path is wrong. It should be `gitlab.projectId`. I just ran into it

Reply:
Nice find! Do you want me to correct it in this PR (since you mentioned it here)?

Reply:
Yes, that'd be great.